chi/context.py

```diff
@@ -237,8 +252,44 @@ def params():
     keys.extend(list(cfg.CONF[_auth_section(_auth_plugin)].keys()))
     return keys
 
+def list_sites(show: Optional[str] = None) -> List[str]:
```

Comment:
I think we should sort this list by `site_class`, with baremetal being first. I noticed it defaults to CHI@Edge, which may cause errors.